specification/network/resource-manager/Microsoft.Network/2017-08-01/virtualNetwork.json

```diff
@@ -825,6 +826,13 @@
           "$ref": "./routeTable.json#/definitions/RouteTable",
           "description": "The reference of the RouteTable resource."
         },
+        "serviceEndpoints": {
```

Comment:
This's breaking changes, same as here https://github.com/Azure/azure-rest-api-specs/pull/1533